### PR TITLE
Fb chart view

### PR DESCRIPTION
**Pull request by LoginovAS**

### CODE REVIEW

File: src/main/java/org/sbx/file/bo/impl/ESK363FileBO.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+public class ESK363FileBO implements FileBO {
+
+    private static final Logger logger = LogManager.getLogger(ESK363FileBO.class);
+
+    private FileDAO esk363FileDAO;
+
+    public ESK363FileBO(){
+
+    }
+/*
```
Comment (adxfighter) [inline src/main/java/org/sbx/file/bo/impl/ESK363FileBO.java]: Please, don't keep commented code

File: src/main/java/org/sbx/file/bo/impl/ESK363FileBO.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                                                .getFileDAO();
+    }
+*/
+    public void setEsk363FileDAO(FileDAO esk363FileDAO){
+        this.esk363FileDAO = esk363FileDAO;
+    }
+
+    public void load(File file) {
+
+        // setEsk363FileDAO();
```
Comment (adxfighter) [inline src/main/java/org/sbx/file/bo/impl/ESK363FileBO.java]: Remove it

File: src/main/java/org/sbx/file/parser/Parser.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+ */
+public class Parser {
+    private String targetString;
+    private String regExp;
+    private List<String> list;
+
+    private static final Logger logger = LogManager.getLogger(Parser.class);
+    public Date getDateByString(String stringDate, String format){
+        Date date = null;
+        DateFormat dateFormat = new SimpleDateFormat(format); //MM.dd.yyyy HH:mm:ss.SSS
```
Comment (adxfighter) [inline src/main/java/org/sbx/file/parser/Parser.java]: Leave comments above , not from the right.
Try to leave more clear and detailed comments

File: src/main/java/org/sbx/servlet/ESK363RangeServlet.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * Created by aloginov on 01.11.16.
+ */
+public class ESK363RangeServlet extends HttpServlet {
+
+    //private RecordBO esk363RecordBO;
```
Comment (adxfighter) [inline src/main/java/org/sbx/servlet/ESK363RangeServlet.java]: Remove unnecessary comments

File: src/main/java/org/sbx/servlet/LoadESK363DataServlet.java [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+        WebApplicationContext context = WebApplicationContextUtils.getWebApplicationContext(getServletContext());
+
+        RecordBO esk363RecordBO = (ESK363RecordBO) context.getBean(Bean.ESK363_RECORD_BO.toString());
+
+        FileBO esk363FileBO = (ESK363FileBO ) context.getBean(Bean.ESK363_FILE_BO.toString());
+
+        // Debug
+        PrintWriter pw = response.getWriter();
+        if (esk363RecordBO == null)
+            pw.println("esk363RecordBO is NULL.");
```
Comment (adxfighter) [inline src/main/java/org/sbx/servlet/LoadESK363DataServlet.java]: Use DEBUG log level instead

File: src/main/java/org/sbx/servlet/ESK363RangeServlet.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException{
+
+        WebApplicationContext context = WebApplicationContextUtils.getWebApplicationContext(getServletContext());
+
+        RecordBO esk363RecordBO = (ESK363RecordBO) context.getBean(Bean.ESK363_RECORD_BO.toString());
+
+        Date dateFrom = Service.stringToDate(request.getParameter("startDate"), EnumDateFormat.INPUT.getFormat());
+        Date dateTo = Service.stringToDate(request.getParameter("endDate"), EnumDateFormat.INPUT.getFormat());
+
+        ArrayList<ESK363DBRecord> resultList = (ArrayList) esk363RecordBO.findByDateRange(dateFrom, dateTo);
```
Comment (adxfighter) [inline src/main/java/org/sbx/servlet/ESK363RangeServlet.java:40]: Add INFO logging. It helps to debug errors and makes code clear

File: src/main/java/org/sbx/servlet/LoadESK363DataServlet.java [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+import java.util.Date;
+import java.util.List;
+
+
+/**
+ * Created by aloginov on 01.11.16.
+ */
+public class LoadESK363DataServlet extends HttpServlet {
+
+    //private RecordBO esk363RecordBO;
```
Comment (adxfighter) [inline src/main/java/org/sbx/servlet/LoadESK363DataServlet.java]: Remove such comments

File: src/main/java/org/sbx/servlet/LoadESK363DataServlet.java [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+        ESK363DBRecord esk363DBRecord = null;
+
+        for (File file: esk363FileBO.getFiles()){
+            esk363FileBO.load(file);
+            list.addAll(esk363FileBO.getData());
+        }
+
+
+        ESK363RecordBuilder esk363RecordBuilder;
+
```
Comment (adxfighter) [inline src/main/java/org/sbx/servlet/LoadESK363DataServlet.java:78]: Add INFO logging

File: src/main/webapp/WEB-INF/web.xml [hunk 9]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<web-app xmlns="http://java.sun.com/xml/ns/j2ee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee/web-app_2_4.xsd"
+         Version="2.4">
+
+    <servlet>
+        <servlet-name>esk363RangeServlet</servlet-name>
+        <servlet-class>org.sbx.servlet.ESK363RangeServlet</servlet-class>
+        <!--init-param>
```
Comment (adxfighter) [inline src/main/webapp/WEB-INF/web.xml]: remove

File: src/main/java/org/sbx/file/bo/impl/ESK363FileBO.java (hunk 1 above)
Comment (LoginovAS) [reply src/main/java/org/sbx/file/bo/impl/ESK363FileBO.java]: Fixed

File: src/main/java/org/sbx/file/bo/impl/ESK363FileBO.java (hunk 2 above)
Comment (LoginovAS) [reply src/main/java/org/sbx/file/bo/impl/ESK363FileBO.java]: Removed

File: src/main/java/org/sbx/servlet/LoadESK363DataServlet.java (hunk 7 above)
Comment (LoginovAS) [reply src/main/java/org/sbx/servlet/LoadESK363DataServlet.java]: Fixed

File: src/main/java/org/sbx/servlet/LoadESK363DataServlet.java (hunk 5 above)
Comment (LoginovAS) [reply src/main/java/org/sbx/servlet/LoadESK363DataServlet.java]: Fixed

File: src/main/webapp/WEB-INF/web.xml (hunk 9 above)
Comment (LoginovAS) [reply src/main/webapp/WEB-INF/web.xml]: Fixed.

**Pull request merged**